### PR TITLE
added menu bar for help and fixed text sizings

### LINKED ISSUE #160: Improvements of the GUI
## User story
1. As a user
2. I want / need an Intuitiv and understandable 
4. So that I can do the tasks easily and fastly reuse the computed difference video. 

## Acceptance criteria
* Windows: the file selection explorer should be a normal optimized explorer with a moder look e.g. 
image
* There should be info buttons, which open a small info window showing short, but beneficial instruction and provide more information about the underlying functionality, e.g. 
Info_button
* The window should contain the the program name and version in the top window frame, e.g. 
image
* There should be more buttons: 
? -> GitHub Project Page: By clicking on it, he GItHub Project Page is opened in the browser  
? -> Helpt: By clicking on it, he documentation is opened in the browser  
e.g. 
image
* The paths of the two input videos should be more clearly visible! The size of the font should not depend on the length of the path! e.g.image

## Definition of done (DoD)
* Create feature branch
* Implement the functionality
* Opened pull request and assigned reviewer
* If necessary: required changes are implemented
* Delete feature branch
* If necessary: Component Test has been written
* Code review has been completed and code has been merged

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

### PR DESCRIPTION
**Pull request by a-miscellaneous**
- [x] menu bar for help
- [x] text sizings
closes #160

### CODE REVIEW

**Comment by a-miscellaneous** [comment]
figuring out trimming took too long

File: GUI/src/main/kotlin/ui/screens/SelectVideoScreen.kt [hunk 1]
```diff
@@ -94,3 +104,38 @@ fun RowScope.AdvancedSettingsButton(state: MutableState<AppState>) {
...
+        Text("?", fontSize = MaterialTheme.typography.body2.fontSize)
+    }
+
+    DropdownMenu(
+        modifier = modifier.padding(8.dp),
+        expanded = expanded,
+        onDismissRequest = { expanded = false },
+    ) {
+        hyperlinkDropdownMenuItem("Project Page", "https://github.com/amosproj/amos2023ws03-gui-frame-diff")
+        hyperlinkDropdownMenuItem("Help", "https://github.com/amosproj/amos2023ws03-gui-frame-diff/blob/main/README.md")
```
Comment (fs3itz) [inline GUI/src/main/kotlin/ui/screens/SelectVideoScreen.kt:124]: i would link to the wiki as it includes some more documentation aswell

**Review by fs3itz — COMMENTED** [review]
i think we should also add the menubar to the diffscreen

File: GUI/src/main/kotlin/ui/screens/SelectVideoScreen.kt (hunk 1 above)
Comment (a-miscellaneous) [reply GUI/src/main/kotlin/ui/screens/SelectVideoScreen.kt]: outdated

**Review by fs3itz — APPROVED** [review]
looks good

**Pull request merged**